platform/osx/detect.py

```diff
@@ -188,12 +182,10 @@ def configure(env):
     )
     env.Append(LIBS=["pthread", "z"])
 
-    env.Append(CPPDEFINES=["VULKAN_ENABLED"])
-    env.Append(LINKFLAGS=["-framework", "Metal", "-framework", "QuartzCore", "-framework", "IOSurface"])
-    if env["use_static_mvk"]:
-        env.Append(LINKFLAGS=["-L$VULKAN_SDK_PATH/MoltenVK/MoltenVK.xcframework/macos-arm64_x86_64/", "-lMoltenVK"])
-        env["builtin_vulkan"] = False
-    elif not env["builtin_vulkan"]:
-        env.Append(LIBS=["vulkan"])
+    if env["vulkan"]:
+        env.Append(CPPDEFINES=["VULKAN_ENABLED"])
+        env.Append(LINKFLAGS=["-framework", "Metal", "-framework", "QuartzCore", "-framework", "IOSurface"])
+        if not env["use_volk"]:
+            env.Append(LINKFLAGS=["-L$VULKAN_SDK_PATH/MoltenVK/MoltenVK.xcframework/macos-arm64_x86_64/", "-lMoltenVK"])
```

Comment:
So the new default for macOS is to use volk, and if disabled it's to use static MoltenVK IIUC, with no option anymore for dynamic MoltenVK. That works for me :+1: 

How does volk handle MoltenVK? Does it find it from the Vulkan SDK if installed, and from our bundled `.xcframework` otherwise?

Reply:
volk will load try loading `libvulkan.1.dylib` (which is installed as part of SDK), if it's not found it will try `libMoltenVK.dylib`.

xcframework if only for static linking only.